<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMainW</class>
 <widget class="QMainWindow" name="StudentMainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>281</width>
      <height>461</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>学籍信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>考试信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>课程安排表</string>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>341</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>学籍信息</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>考试信息</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>296</width>
        <height>236</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_stuID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>学生账号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_stuID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
